include/LLGL/Buffer.h

```diff
@@ -58,6 +58,15 @@ class LLGL_EXPORT Buffer : public Resource
         */
         virtual BufferDescriptor GetDesc() const = 0;
 
+        /**
+        \brief Sets the vertex attributes of this buffer.
+        \param[in] vertexAttribs Array view of vertex attributes to set.
+        \remarks This function is only used for vertex buffers and ignored if \c bindFlags does not contain the BindFlags::VertexBuffer bit.
+        \see BindFlags::VertexBuffer
+        \see VertexShaderAttributes::inputAttribs
+        */
+        virtual void SetVertexAttribs(const ArrayView<VertexAttribute>& vertexAttribs) {};
```

Comment:
I assume this only had a default implementation for debugging purposes, but for your final PR, pelase make this a pure virtual function. This means you'll also have to provide an implementation in the DebugLayer (i.e. `DbgBuffer`), Null backend (i.e. `NullBuffer`), and Metal backend (i.e. `MTBuffer`).